notebooks/plate_scale_plots.ipynb

```diff
@@ -0,0 +1,1184 @@
+    "import lsst.jointcal.cameraGeometry\n",
+    "import lsst.obs.hsc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "turned-original",
+   "metadata": {},
+   "source": [
+    "## Read data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "large-bones",
+   "metadata": {},
+   "source": [
+    "For this demo, we use the jointcal output files that are included for the tests, and create an HSC camera directly from obs_subaru. If you have a butler repo with jointcal output, read in the WCSs you want with the `jointcal_wcs` dataset type, make a list out of them to pass to `cameraGeometry.CameraModel`, and a 1-1 matched list with the detector ids for each of those WCSs. You also need to get the camera object from the butler (`camera` dataset type)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "potential-roommate",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 49 is near the center of the focal plane\n",
+    "detector1 = 49\n",
+    "# 37 is on the west side of the focal plane\n",
+    "detector2 = 37\n",
+    "# 103 is on the NW edge of the focal plane and rotated 90 degrees\n",
+    "detector3 = 103\n",
+    "# 90 is on the NE corner of the focal plane\n",
+    "detector4 = 90\n",
+    "\n",
+    "cameraPath = os.path.join(lsst.utils.getPackageDir(\"obs_subaru\"), \"hsc/camera\")\n",
+    "cameraConfig = lsst.afw.cameraGeom.CameraConfig()\n",
+    "cameraConfig.load(os.path.join(cameraPath, \"camera.py\"))\n",
+    "camera = lsst.afw.cameraGeom.makeCameraFromPath(cameraConfig, cameraPath, lambda x: x,\n",
+    "                                                     lsst.obs.hsc.HscPupilFactory)\n",
+    "maxFocalPlaneRadius = camera.computeMaxFocalPlaneRadius()\n",
+    "\n",
+    "path = \"../tests/data/output/HSC-R/9615/\"\n",
+    "\n",
+    "wcsList = []\n",
+    "detectors = []\n",
+    "for detector, visit in itertools.product([detector1, detector2, detector3],\n",
+    "                                  [28976, 23924, 23900]):\n",
+    "        wcsList.append(SkyWcs.readFits(os.path.join(path, f\"jointcal_wcs-{visit:07}-{detector:03}.fits\")))\n",
+    "        detectors.append(detector)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ahead-consent",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = lsst.jointcal.cameraGeometry.CameraModel(wcsList, detectors, camera)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "threaded-shield",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model.computePixelScale()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "permanent-joseph",
+   "metadata": {},
+   "source": [
+    "## Make some plots"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "boolean-latvia",
+   "metadata": {},
+   "source": [
+    "To see the differences between the curves and how they overlap with the standard deviation, you have to zoom in a lot."
```

Comment:
It'd be helpful to call out the residual plot here.  Specifically, that you're finding some small differences between the nominal camera distortion model and the actual WCS fits.